### DISCUSSION TITLE
Where is the 3.5 Version?

### DISCUSSION THREAD

**Discussion by SwishyBro473** [Techokami/SonicWorldsNext#56] [Q&A]
I go to install it and it says it was made in Godot 4, wheres the 3 / 3.5 version?

**Comment by BlipShift2005** [comment]
I guess the link in the README.md is incorrect?
The branch is named [Sonic-Worlds-Next-Godot-3](https://github.com/Techokami/SonicWorldsNext/tree/Sonic-Worlds-Next-Godot-3)

**Comment by DimensionWarped** [comment]
Right, so you just need to clone the repo and then check out the Sonic-Worlds-Next-Godot-3 branch.

Or if you aren't actually using git (you should be) to check it out but using the download zip function instead, you can go here and then choose download zip.

https://github.com/Techokami/SonicWorldsNext/tree/Sonic-Worlds-Next-Godot-3
